packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/media/ImageStreamReaderUtilsTest.java

```diff
@@ -25,52 +22,10 @@ public void setUp() {
     this.imageStreamReaderUtils = new ImageStreamReaderUtils();
   }
 
-  Image getImage(int imageWidth, int imageHeight, int padding) {
-    int rowStride = imageWidth + padding;
-
-    int ySize = (rowStride * imageHeight) - padding;
-    int uSize = (ySize / 2) - (padding / 2);
-    int vSize = uSize;
-
-    // Mock YUV image
-    Image mockImage = mock(Image.class);
-    when(mockImage.getWidth()).thenReturn(imageWidth);
-    when(mockImage.getHeight()).thenReturn(imageHeight);
-    when(mockImage.getFormat()).thenReturn(ImageFormat.YUV_420_888);
-
-    // Mock planes. YUV images have 3 planes (Y, U, V).
-    Image.Plane planeY = mock(Image.Plane.class);
-    Image.Plane planeU = mock(Image.Plane.class);
-    Image.Plane planeV = mock(Image.Plane.class);
-
-    // Y plane is width*height
-    // Row stride is generally == width but when there is padding it will
-    // be larger.
-    // Here we are adding 256 padding.
-    when(planeY.getBuffer()).thenReturn(ByteBuffer.allocate(ySize));
-    when(planeY.getRowStride()).thenReturn(rowStride);
-    when(planeY.getPixelStride()).thenReturn(1);
-
-    // U and V planes are always the same sizes/values.
-    // https://developer.android.com/reference/android/graphics/ImageFormat#YUV_420_888
-    when(planeU.getBuffer()).thenReturn(ByteBuffer.allocate(uSize));
-    when(planeV.getBuffer()).thenReturn(ByteBuffer.allocate(vSize));
-    when(planeU.getRowStride()).thenReturn(rowStride);
-    when(planeV.getRowStride()).thenReturn(rowStride);
-    when(planeU.getPixelStride()).thenReturn(2);
-    when(planeV.getPixelStride()).thenReturn(2);
-
-    // Add planes to image
-    Image.Plane[] planes = {planeY, planeU, planeV};
-    when(mockImage.getPlanes()).thenReturn(planes);
-
-    return mockImage;
-  }
-
   /** Ensure that passing in an image with padding returns one without padding */
   @Test
   public void yuv420ThreePlanesToNV21_trimsPaddingWhenPresent() {
-    Image mockImage = getImage(160, 120, 16);
+    Image mockImage = ImageStreamReaderTest.getImage(160, 120, 16, ImageFormat.YUV_420_888);
```

Comment:
Nit: seems non-intuitive for test files to call between each other. Can you create an `ImageStreamReaderTestUtils.java` or something like that and put it there?